stack-graphs/src/json.rs

```diff
@@ -230,6 +230,14 @@ impl<'a> StackGraph {
     pub fn to_json(&'a self, f: &'a dyn Filter) -> JsonStackGraph {
         JsonStackGraph(self, f)
     }
+
+    pub fn to_serializable(&self) -> crate::serde::StackGraph {
+        self.to_serializable_with_filter(&NoFilter)
+    }
+
+    pub fn to_serializable_with_filter(&self, f: &'a dyn Filter) -> crate::serde::StackGraph {
```

Comment:
What do you think about calling this `to_serializable_filter`, similar to stdlib methods such as https://doc.rust-lang.org/std/vec/struct.Vec.html#method.drain_filter?

Reply:
agreed, that is a lot better than the `with_` prefix.